.github/workflows/ci.yml

```diff
@@ -45,3 +54,4 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          
```

Comment:
@Praveenraj-K also, please set your IDE to add null-termination at the end of the file.

Here's the setting in VSCode:
![image](https://github.com/user-attachments/assets/e83eb594-4a6f-4dc3-8755-3d1c82e45e6f)

Reply:
I find the diff confusing. Here I thought he added a new line due to the green. I even asked copilot and it agreed!

Reply:
I agree, it's a bit strange. But it looks like he added an extra non-terminated blank line. Here's the git diff output after I deleted that blank line in vim.
```
14:26  $ git diff
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index b059386..5314d1c 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -54,4 +54,3 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          
\ No newline at end of file
```

Reply:
> I find the diff confusing. Here I thought he added a new line due to the green. I even asked copilot and it agreed!

Thanks for @briskt  updated the setting in the vs code.